examples/07_advanced/plot_beta_series.py

```diff
@@ -0,0 +1,191 @@
+"""
+Beta-Series Modeling for Task-Based Functional Connectivity and Decoding
+========================================================================
+This example shows how to run :term:`GLM`s for beta series models, which are a
+common modeling approach for a variety of analyses of task-based fMRI
+data with an event-related task design, including functional connectivity,
+decoding, and representational similarity analysis.
+
+Beta series models fit trial-wise conditions, which allow users to create
+"time series" of these trial-wise maps, which can be substituted for the
+typical time series used in resting-state functional connectivity analyses.
+Generally, these models are most useful for event-related task designs,
+while other modeling approaches (e.g., PPI) tend to perform better in block
+designs, depending on the type of analysis.
+See [Cisler, Bush, & Steele (2014)]
+(https://doi.org/10.1016/j.neuroimage.2013.09.018)
+for more information about this, in the context of functional connectivity
+analyses.
+
+Two of the most well-known beta series modeling methods are
+Least Squares- All (LSA) and Least Squares- Separate (LSS).
+In LSA, a single :term:`GLM` is run, in which each trial of each condition of
+interest is separated out into its own condition within the design matrix.
+In LSS, each trial of each condition of interest has its own :term:`GLM`,
+in which the targeted trial receives its own column within the design matrix,
+but everything else remains the ssame as the standard model.
+Trials are then looped across, and many GLMs are fitted,
+with the parameter estimate map extracted from each GLM to build the LSS beta
+series.
+"""
+from nilearn.glm.first_level import FirstLevelModel
+from nilearn import image, masking, plotting
+
+##############################################################################
+# Prepare data and analysis parameters
+# -------------------------------------
+# Prepare the data.
+from nilearn.datasets import fetch_language_localizer_demo_dataset
+
+data_dir, _ = fetch_language_localizer_demo_dataset()
+
+from nilearn.glm.first_level import first_level_from_bids
+
+models, models_run_imgs, events_dfs, models_confounds = \
+    first_level_from_bids(
+        data_dir,
+        'languagelocalizer',
+        img_filters=[('desc', 'preproc')],
+    )
+
```

Comment:
Here you probably want to epxlain that you(re extracting data from a single subject.